snark-utils/benches/io.rs

```diff
@@ -19,25 +17,18 @@ use test_helpers::*;
 ///   no computational bottleneck, hence the parallelism ends up introducing more overhead than benefit
 fn read<C: AffineCurve>(c: &mut Criterion, el_type: &str) {
     let mut group = c.benchmark_group(format!("read_batched_{}", el_type));
-    let els = (1..14)
+    group.sample_size(10);
+    let els = (8..11)
         .map(|i| 2u32.pow(i) as usize)
         .collect::<Vec<usize>>();
 
-    for compression in &[UseCompression::Yes, UseCompression::No] {
+    for compression in &[UseCompression::Yes] {
+        // , UseCompression::No] {
```

Comment:
Is this intentional?

Reply:
Oops, nope, but I'll fix in my follow up PR.